### DISCUSSION TITLE
Long Term Timelapse QR

### DISCUSSION THREAD

**Discussion by Othman89** [gopro/labs#818] [Q&A]
Buongiorno a tutti, scrivo in italiano perché così riesco a descrivere meglio il mio quesito, non sono un informatico, non esporto di codici, per questo sono a chiedervi una mano, ho installato il firmware sulla mio go pro 12 e dovrei fare un timelapse di 1 mese, ho visto che posso creare dei qr con la funzione daily timelapse , ma le fa in formato video, vorrei poterlo fare con la modalità foto, la Gopro sarà alimentata direttamente con la corrente e non userò batterie, ho comprato un router esterno, con una sim dati per poter usare il cloud di gopro, ma non l'ho mai usato, avete qualche soluzione anche per inserire nel qr la funzione cloud?

**Comment by dnewman-gpsw** [comment]
You photo timelapse you much change the preset to output photos. 

'''
Time/Night-Lapse modes: The camera defaults to having presets for only one time-lapse mode and one night-lapse mode, yet both modes have two configurations (video or photo.) For all QR Code mode commands to work, a preset is needed for every mode you wish to use. e.g. If you only have a preset for night-lapse video (NLV), the QR command for NLV will work, but will not for night-lapse photo (NLP). To solve this, create a new preset for NLP, and both video and photo night-lapse commands will work.
'''

Once you have the require preset, the Daily Timelapse example can be altered.

from  ```!08:00NmTp.60!S!3600E!1R```  to ```!08:00NmTPp.60!S!3600E!1R```

Just adding the **P** to make it Timelapse Photo.

Warning. 5G access points are typically very slow on upload, maybe too slow to reliable upload a daily photo timelapse as JPEGs are not very efficient. The video timelapse will be more reliable.

**Comment by Othman89** [comment]
Grazie mille per la risposta,

Oggi mentre ho svolto dei lavori per la costruzione del timelapse, mi hanno dato più informazioni 

Il lavoro si svolgerà dal lunedì al venerdì dalle ore 8:00 alle ore 17:30, vorrei seguire il tuo consiglio è farlo diventare video e non foto, con un frame ogni 5 minuti, togliendo il timer della gopro, è possibile impostare un timelapse come questo? 


Grazie mille

**Reply by dnewman-gpsw** [reply]
Yes it is possible.  First experiment with https://gopro.github.io/labs/control/dailytl/

You make it skip the weekend, you will need to do some coding within https://gopro.github.io/labs/build/
